src/main/java/study/CalculationType.java

```diff
@@ -0,0 +1,31 @@
+package study;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum CalculationType {
+    PLUS("+", (a, b) -> a + b),
+    MINUS("-", (a, b) -> a - b),
+    MULTYPLY("*", (a, b) -> a * b),
+    DIVIDE("/", (a, b) -> a / b);
+
+    private String calculationString;
+    public Calculation calculation;
+
+
+    CalculationType(String calculationString, Calculation calculation) {
+        this.calculationString = calculationString;
+        this.calculation = calculation;
+    }
+
+
+    public static CalculationType checkCalculationType(String inputString) {
+        Optional<CalculationType> cType = Arrays.stream(CalculationType.values()).filter(calculationType -> calculationType.calculationString.equals(inputString)).findFirst();
+        return cType.orElseThrow(IllegalArgumentException::new);
+    }
+
+    public static int calculate(CalculationType calculator, int a, int b) {
+        return calculator.calculation.calculate(a, b);
+    }
```

Comment:
```suggestion
  public static int execute (char operator, int x, int y) {
    return Arrays.stream(Operation.values())
            .filter(v -> v.operator == operator)
            .findFirst()
            .orElseThrow(() -> new IllegalArgumentException(CalculatorError.INVALID_OPERATOR))
            .operation.apply(x, y);
  }
```
이렇게 표현하면 어떨까요?

그리고 `Arrays.stream(CalculationType.values()).filter(calculationType -> calculationType.calculationString.equals(inputString)).findFirst();`
이렇게 dot(`.`)이 많을 경우엔 줄바꿈을 해주세요! 보통 한 줄에 한 개를 권장합니다.